baram/iam_manager.py

```diff
@@ -27,6 +27,19 @@ def get_role_arn(self, role_name):
         '''
         return self.get_role(role_name)['Arn']
 
+    def list_roles(self, max_result: int = 1000):
+        """
+        Lists all roles
+        :param max_result: max number of results (max=1000)
+        :return:
+        """
+        roles = self.cli.list_roles(MaxItems=max_result)
+        result = roles['Roles']
+        while 'Marker' in roles:
```

Comment:
what if data structure contains like below? does your logic run as intended?

roles 1 : has marker
roles 2 : doesn't have marker
roles 3 : has marker

Reply:
Marker is for pagination if total result exceeds max result number limitation, which is 1000. If the result is truncated, we use marker to get next page and it will be continued until there's no marker, meaning result is not truncated anymore.
So the case you mentioned above will not happen.

Reply:
can you merge 36~38 lines?